src/app/modules/layout/components/admin-layout/admin-layout.component.ts

```diff
@@ -113,6 +114,8 @@ export class AdminLayoutComponent implements OnInit, AfterViewInit {
     this.currentTheme = this.themeService.currentTheme().name;
     const navigationHold = document.getElementById('scroll-area');
 
+    this.store$.dispatch(loadFailoverLicensedStatus());
```

Comment:
Afaik, ngrx generally recommends using actions as events (e.g. somethingHappened) instead of commands (doSomething).
There is already a `adminUiInitialized` that you can rely on to load HA status.